app/src/main/java/com/github/polybooks/core/Books.kt

```diff
@@ -29,7 +29,7 @@ data class Book(
     val language : String?,
     val publisher : String?,
     val publishDate : Timestamp?,
-    val format : String?
+    val format : String?,
```

Comment:
Remove the last comma

Reply:
I think this doesn't change anything, no?

Reply:
It doesn't change functionality, but I think we agreed on our coding guidelines to remove those trailing commas, so I'll remove it.